content/03.methods.md

```diff
@@ -40,21 +41,23 @@ All results were output to json format for further downstream processing.
 
 #### _Springer Nature_ API
 
-_Springer Nature_ was chosen over other publishers for multiple reasons: 1) it is a large publisher, second only to Elsevier; 2) it covers multiple subjects, in contrast to PubMed; 3) its API has a large daily query limit (5000/day); and 4) it provided more author affiliation information than found in Elsevier. 
+_Springer Nature_ was chosen over other publishers and search engines for multiple reasons: 1) ease of use; 2) it is a large publisher, second only to Elsevier; 3) it covers diverse subjects, in contrast to PubMed, which focuses on the biomedical and life sciences literature; 4) its API has a large daily query limit (5000/day); and 5) it provided more author affiliation information than found in Elsevier. 
```

Comment:
Hypothetical: would the manuscript be clearer if we removed the _Springer Nature_ corpus?

Reply:
it would be clearer, but I think including it as a comparator is worth the extra possible confusion to the reader.